index.bs

```diff
@@ -950,6 +950,24 @@ spec: ecma-262; urlPrefix: http://tc39.github.io/ecma262/
       </tr>
     </tbody>
   </table>
+
+  APIs for creating and navigating browsing contexts by name {#patch-window-apis}
+  -------------------------------------------------------------------------------
+
+  Modify the definition of <a spec=HTML>script-closable</a> to prevent portals from closing
+  themselves pre-activation:
```

Comment:
Maybe we should word as closing when in a portal browsing context, as opposed to pre-activation? Mainly because a portal could be adopted, which may count as post-activation, but we would still want to prevent it from self-closing.

Reply:
Yep, good call; I was trying to be informal in the informal intro, and I ended up being confusing.